src/nspawn/nspawn.c

```diff
@@ -30,6 +30,7 @@
 #ifdef HAVE_SELINUX
 #include <selinux/selinux.h>
 #endif
+#include "selinux-util.h"
```

Comment:
This line should be there https://github.com/rhatdan/systemd/blob/selinux/src/nspawn/nspawn.c#L91

https://github.com/systemd/systemd/blob/master/CODING_STYLE#L301
> please always include the
  headers of external components first (these are all headers enclosed
  in <>), followed by our own exported headers (usually everything
  that's prefixed by "sd-"), and then followed by internal headers.
  Furthermore, in all three groups, order all includes alphabetically
  so duplicate includes can easily be detected.